packages/block-editor/src/components/block-heading-level-dropdown/stories/index.story.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import HeadingLevelDropdown from '../';
+
+const meta = {
+	title: 'BlockEditor/HeadingLevelDropdown',
+	component: HeadingLevelDropdown,
```

Comment:
```suggestion
	component: HeadingLevelDropdown,
	parameters: {
		docs: {
			canvas: { sourceState: 'shown' },
			description: {
				component:
					'Dropdown for selecting a heading level (1 through 6) or paragraph (0).',
			},
		},
	},
```

- Let's show the source panel: https://storybook.js.org/docs/api/doc-blocks/doc-block-canvas#sourcestate
- Define the component description explicitly, otherwise the description will be a bit hard to read:
  ![image](https://github.com/user-attachments/assets/d57b3d21-5206-438d-a608-eea9939776d1)